Description: replace pastebin.com with paste.debian.net as default
 until breakage from API changes has been fixed upstream
Author: Rolf Leggewie <foss@rolf.leggewie.biz>
Forwarded: not-needed
Last-Update: <2012-05-20>

--- pastebinit-1.3.orig/pastebinit
+++ pastebinit-1.3/pastebinit
@@ -17,7 +17,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
-defaultPB = "http://pastebin.com" #Default pastebin
+defaultPB = "http://paste.debian.net" #Default pastebin
 try:
     import lsb_release
     release = lsb_release.get_distro_information()['ID'].lower()
